src/main/kotlin/no/nav/dagpenger/regel/api/db/PostgresSubsumsjonStore.kt

```diff
@@ -133,6 +134,7 @@ internal class PostgresSubsumsjonStore(private val dataSource: DataSource) : Sub
                         antallBarn = row.intOrNull("antall_barn"),
                         manueltGrunnlag = row.intOrNull("manuelt_grunnlag"),
                         inntektsId = row.stringOrNull("inntekts_id"),
+                        sikringsordningLærling = row.boolean("sikringsordning_laerling"),
```

Comment:
Stusser litt over denne. Finnes ingen `booleanOrNull`. Kommer denne til å plage oss?

Reply:
mm, ja det er jo litt rart med nullable boolske verdier, mulig en må gjøre `row.stringOrNull("sikringsordning_laerling")?.let { Boolean.valueOf(it) }` elns

Reply:
`sikringsordning_laerling` er litt rart navn når jeg ser det?

Reply:
Ja, enig i at det er rart navn. Ville få frem at det ikke bare er for lærlinger på generell basis

Reply:
Ser forresten at apiet returnerer `false` hvis verdien er `null`, så det går greit å kalle på `boolean()` her